### ISSUE TITLE
[question]: do we really need playwright library or not?

### ISSUE THREAD

**Issue by stevez** [abhinaba-ghosh/playwright-lighthouse#56]
I found if I use the following code, then the playwright library can be removed:

 test("Ligthouse performance test", async ({playwright}) => {
      const browser = await playwright.chromium.launch({
          args: ['--remote-debugging-port=9222'],
        });
  
      const page = await browser.newPage();
      await page.goto('https://www.google.ca');

  ....

**Comment by abhinaba-ghosh** [comment]
Are you looking to connect the library with an existing playwright connection? If yes, we can check the possibility to do so.

**Comment by alphonse92** [comment]
Not at all, but you can open a new browser to do that. But if you want to use the same playwright browser you could add the remote debugging port into playwright configuration:


![Screenshot 2024-08-06 at 10 12 54 AM](https://github.com/user-attachments/assets/0be46bb3-4cc8-44e8-a0b2-e254e2237ae3)

**Comment by vincerubinetti** [comment]
I found that setting the `--remote-debugging-port` globally in the config messed up my other tests.

I'm finding when I follow the exact example in the readme, I get the following `Test timeout of 30000ms exceeded.` `browserType.launch` error on _some_ of my Lighthouse tests, consistently.

<img width="1219" alt="Screenshot 2024-10-29 at 3 09 18 PM" src="https://github.com/user-attachments/assets/2603c57d-eed6-4e50-8b07-f53ce573af7f">

Edit: turning off `fullyParallel` seemed to fix it.

Another edit: Sorry, I missed the ["Using with the Playwright test-runner"](https://github.com/abhinaba-ghosh/playwright-lighthouse?tab=readme-ov-file#usage-with-playwright-test-runner) part of the readme, where it shows how to support parallel runs. Not to nitpick but shouldn't this section be near the top of the readme? Isn't this what most people are using this library for?
